initrd/bin/cbfs-init

```diff
@@ -7,16 +7,30 @@ if [ -z "$CONFIG_PCR" ]; then
 	CONFIG_PCR=7
 fi
 
+if [ "$CONFIG_BOARD" = "talos-2_server" ] || [ "$CONFIG_BOARD" = "talos-2_workstation" ]; then
+	# We're running before /tmp/config exists, but flash.sh needs it, so generate its initial version
+	combine_configs
+
+	echo "Reading ROM..."
```

Comment:
@SergiiDmytruk That won't fly.

Even if flashrom was meant to read only once to /tmp (major TOCTOU for everything else that relies on that backup later on) this would mean adding ~30 seconds at each boot?!
```

~ # time flashrom -p linux_mtd -r /tmp/tst
flashrom  on Linux 5.5.0-openpower1 (ppc64le)
flashrom is free software, get the source code at https://flashrom.org

Using clock_gettime for delay loops (clk_id: 1, resolution: 1ns).
Opened /dev/mtd0 successfully
Found Programmer flash chip "Opaque flash chip" (65536 kB, Programmer-specific) on linux_mtd.
Reading flash... done.
real	0m 26.32s
user	0m 0.08s
sys	0m 25.91s
```

Since flashtools is already modified, why cbfs cannot be modified to be able to search for base and not rely on flashrom and then pnor to get cbfs content?